config/locales/telephony/en.yml

```diff
@@ -21,8 +21,8 @@ en:
       voice: Hello! Your %{app_name} one-time code is, %{code}. Your one-time code is,
         %{code}. Again, your one-time code is %{code}. This code expires in
         %{expiration} minutes.
-    doc_auth_link: "%{link} You've requested to verify your identity on a mobile
-      phone.  Please take a photo of your state issued ID."
+    doc_auth_link: |-
+      %{app_name}: %{link} You're verifying your identity to access %{service_provider}. Take a photo of your ID to continue.
```

Comment:
I've seen a few other takes on naming the interpolated value to clarify that `APP_NAME` is possibly inserted, like `sp_or_app_name` or `service_provider_or_app_name`. Do you think it'd be worth doing something similar here?

Reply:
Yep. Good suggestion!